src/header/header.scss

```diff
@@ -403,7 +367,8 @@
 
```

Comment:
COuld you double check those rules on hover above? I think now it is working differently than before (maybe better?). Now split button is hoverable when active, before it was not.
And simple buttons have no effect on hover..

Reply:
Check again, my latest push I believe fixed this.